### DISCUSSION TITLE
When will the new version (version 1.0?) be released?

### DISCUSSION THREAD

**Discussion by zkk960317** [Ferrite-FEM/Ferrite.jl#896] [General]
Firstly, thanks to the contributors for their hard work. However, Ferrite hasn't been updated in a while, and users need the confidence that the package is still in continuous development.

**Comment by termi-official** [comment]
Hi @zkk960317 , we are actively working on it and currently in the process of wrapping up some planned changes towards 1.0 in tandem with some major new features, for example
* support for H(div) and H(curl) conforming elements
* GPU examples
* adaptive mesh refinement
* more dynamic sparsity patterns and matrix formats
* interface elements

Since we wanted to bundle all possibly breaking changes in the next release we decided to delay release 1.0. I think there are PRs open for everything open todo, but we might a bit more time to polish and test them properly. I hope that explains the situation.
